src/bun.js/bindings/BundlerMetafile.cpp

```
@@ -0,0 +1,86 @@
+    RETURN_IF_EXCEPTION(scope, {});
+
+    if (parsedValue.isUndefined()) {
+        // JSON parse failed, return undefined
+        return JSValue::encode(jsUndefined());
+    }
+
+    // Memoize: replace the getter with the parsed value using putDirect
+    thisObject->putDirect(vm, Identifier::fromString(vm, metafilePropertyName), parsedValue, 0);
+
+    // Delete the raw string property since we no longer need it
+    thisObject->deleteProperty(globalObject, Identifier::fromString(vm, metafileStringPropertyName));
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)